plugins/postcss-progressive-custom-properties/README.md

```diff
@@ -0,0 +1,110 @@
+# PostCSS Progressive Custom Properties [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS" width="90" height="90" align="right">][postcss]
+
+[![NPM Version][npm-img]][npm-url]
+[![Build Status][cli-img]][cli-url]
+[<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Progressive Custom Properties] is a utility plugin to correctly declare Custom Property fallbacks and enhancements.
+
+⚠️ It is not intended to be used directly by stylesheet authors.
+Meant to be included in other PostCSS plugins that provide CSS value transforms as fallbacks.
+
+[Custom Properties are not discarded like regular declarations when invalid.](https://drafts.csswg.org/css-variables/#invalid-variables)
+This makes it tricky to provide fallback values for older browsers.
+
+The solution is to wrap Custom Property declarations in an `@supports` rule.
```

Comment:
Should we make any notes about `@support` support? See [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports#browser_compatibility)

Not for us since I think features we'll be using this for are ahead of `@support` and they just fail silently, but rather for any plugin author that might be interested in this?

Reply:
> Not for us since I think features we'll be using this for are ahead of @support and they just fail silently, but rather for any plugin author that might be interested in this?

exactly :)
I wouldn't specifically mention this.